doc/developer/BFD_Scale_Enhancements.md

```diff
@@ -0,0 +1,209 @@
+# Feature Name
+BFD Scale Enhancement
+
+# High Level Design Document
+# Table of Contents
+- [1 Feature Overview](#1-Feature-Overview)
+    - [1.1 Target Deployment Use Cases](#11-Target-Deployment-Use-Cases)
+    - [1.2 Requirements](#12-Requirements)
+    - [1.3 Design Overview](#13-Design-Overview)
+        - [1.3.1 Basic Approach](#131-Basic-Approach)
+- [2 Design](#3-Design)
+    - [2.1 Overview](#31-Overview)
+        - [2.1.1 Problem statement](#211-Problem-statement)
+        - [2.1.2 Solutions to the identified issues](#212-Solutions-to-the-identified-issues)
+        - [2.1.3 Service and Docker Management](#213-Service-and-Docker-Management)
+        - [2.1.4 Packet Handling](#214-Packet-Handling)
+- [3 Serviceability and Debug](#3-Serviceability-and-Debug)
+   - [3.1 Debug CLI](#3.1-Debug-CLI)
+   - [3.2 LOG](#3.2-LOG)
+
+# List of Tables
+[Table 1: Abbreviations](#table-1-Abbreviations)
+
+# Revision
+| Rev |     Date    |       Author       | Change Description                |
+|:---:|:-----------:|:------------------:|-----------------------------------|
+| 0.1 | 01/27/2021  |   Sumit Agarwal    | Initial version                   |
+
+# About this Manual
+This document provides comprehensive functional and design information about the *BFD scale support* feature implementation in SONiC.
```

Comment:
This is not sonic.

Reply:
These enhancements are results of issues observed while using FRR in SONIC, i am positing Design doc first, code will follow.
Since these enhancements are generic in nature they will be helpful for usage of FRR BFD in any platform.

Reply:
As I stated this is not sonic.  Please fix the doc

Reply:
updated